Dockerfile.dev

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.10-alpine
+RUN apk add --update --no-cache ca-certificates git
+RUN go get github.com/kubernetes-sigs/aws-iam-authenticator/cmd/aws-iam-authenticator
+RUN go get github.com/derekparker/delve/cmd/dlv
+
+FROM alpine:3.6
+RUN apk add --no-cache ca-certificates
```

Comment:
As far as I know, Delve requires `libc6-compat` on alpine to work properly. Did this image work without it?